services/notify.js

```diff
@@ -44,4 +45,19 @@ function asArray(values = []) {
         : [values];
 }
 
-module.exports = { sendNotification }
+const notifyTemplate = async (userInfo, sendAdmin, sendUser) => {
+    // send admin notification
+    if (sendAdmin) {
+        const adminEmails = await getAdminEmails();
+        if (adminEmails && adminEmails.length > 0) await sendAdmin(adminEmails, {});
+        else console.error("Admin email is not found, please check if administrator user existed");
```

Comment:
if admin does not exists, it will not send email to prevent app crash. It also leave an error message.